File: fixtures/dart/get_with_all_numerical_data.dart
import 'package:http/http.dart' as http;

void main() async {
  var headers = {
    'Content-Type': 'application/json',
    'Accept': 'application/json',
  };

  var data = '18233982904';

  var url = Uri.parse('http://198.30.191.00:8309/CurlToNode');
  var res = await http.post(url, headers: headers, body: data);
  if (res.statusCode != 200) throw Exception('http.post error: statusCode= ${res.statusCode}');
  print(res.body);
}

File: test/fixtures/node-http/get_with_all_numerical_data.js
import http from 'http';

const options = {
  hostname: 'localhost:28139',
  path: '/CurlToNode',
  method: 'POST',
  headers: {
    'Content-Type': 'application/json',
    'Accept': 'application/json'
  }
};

const req = http.request(options, function (res) {
  const chunks = [];

  res.on('data', function (chunk) {
    chunks.push(chunk);
  });

  res.on('end', function () {
    const body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.write('18233982904');
req.end();
